frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -11,7 +11,7 @@ import {
   SelfSubjectAccessReviewKind,
   HorizontalPodAutoscalerKind,
 } from '@console/internal/module/k8s';
-import { RevisionModel } from '@console/knative-plugin';
+import { REVISION_KIND } from '@console/knative-plugin';
```

Comment:
we don't need to introduce const for those resources for which we still have static models so it'll like duplicate, we can let use the model for now and we can do clean up as another PR.